iOS/chronreact/AppDelegate.m

```diff
@@ -42,7 +44,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * generated by "Bundle React Native code and images" build step.
    */
 
-//   jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+//  jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
```

Comment:
When you switch to a production build you switch between the two jsCodeLocation declarations so we want to keep this one